src/logstash/scripts/production_locations/rba_id.rb

```
@@ -0,0 +1,21 @@
+
+def filter(event)
+  rba_id = event.get('rba_id_value')
+  rba_id_value = ""
+
+    if rba_id
+      begin
+        parsed = JSON.parse(rba_id)
+        if parsed['raw_value']
+          rba_id_value = parsed['raw_value'].strip
+        end
+      rescue JSON::ParserError => e
```

RuboCop (1.73)

[warning] Useless assignment to variable - `e`.

(Lint/UselessAssignment)